config/projects.yaml

```diff
@@ -284,6 +284,16 @@ ecommerce:
             variants:
                 symfony/symfony: ['4.4']
 
+entity-audit-bundle:
+    excluded_files:
+        - LICENSE
```

Comment:
Shouldn't we update the license?

Reply:
I'm not sure, I prefer to discuss about it before. We can remove this line later if needed